sequencer/src/state_signature/relay_server.rs

```diff
@@ -308,14 +308,6 @@ impl StateRelayServerDataSource for StateRelayServerState {
             },
         };
         let epoch = epoch_from_block_number(block_height, blocks_per_epoch);
-
-        // if epoch is activated and block_height is greater than epoch root, ignore and drop
-        if block_height > self.epoch_start_block.expect("init_genesis wrong")
```

Comment:
can we keep these changes in the file?

Reply:
do you want only this change from main or the whole file>